### PR TITLE
undefined data error

### PR DESCRIPTION
**Pull request by LuisGonzalezJr**
# PR Description
fixed undefined data error

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## Additional Context or Relevant Issues

Please include additional context.

## Issue Link

Please link the original issue URL.

### CODE REVIEW

**Review by AhmedHussein-1 — APPROVED** [review]

**Review by mamohamed252 — DISMISSED** [review]
Please fill out PR template. Approvers make sure PR template is complete before merge.

**Review by InfAurora — APPROVED** [review]
Nice

CarelessCorruptAmazontreeboa-size_restricted

**Pull request merged**
